src/algebraic_geometry/presheafed_space.lean

```diff
@@ -103,22 +103,39 @@ variables {C}
 instance {X Y : PresheafedSpace.{v} C} : has_coe (X ⟶ Y) (X.to_Top ⟶ Y.to_Top) :=
 { coe := λ α, α.f }
 
-@[simp] lemma id_f (X : PresheafedSpace.{v} C) : ((𝟙 X) : X ⟶ X).f = 𝟙 X.to_Top := rfl
+@[simp] lemma id_f (X : PresheafedSpace.{v} C) :
+  ((𝟙 X) : X ⟶ X).f = 𝟙 X.to_Top :=
```

Comment:
I would prefer if the coerced version were the canonical form. Or does this play bad with automation?
```suggestion
  ((𝟙 X) : X ⟶ X).f = 𝟙 (X : Top) :=
```